api/v1beta1/flowcollector_types.go

```diff
@@ -358,6 +358,21 @@ type FlowCollectorFLP struct {
 	// kafkaConsumerBatchSize indicates to the broker the maximum batch size, in bytes, that the consumer will accept. Ignored when not using Kafka. Default: 10MB.
 	KafkaConsumerBatchSize int `json:"kafkaConsumerBatchSize"`
 
+	//+kubebuilder:default:={"flowLog"}
+	// outputRecordTypes is a list of record types to export from ["flowLog","newConnection","heartbeat","endConnection"]
+	// +optional
+	OutputRecordTypes *[]string `json:"outputRecordTypes,omitempty"`
+
+	//+kubebuilder:default:="30s"
+	// +optional
+	// connection update interval is the duration of time to wait between update reports of a connection
+	ConnectionUpdateInterval *metav1.Duration `json:"connectionUpdateInterval,omitempty"`
```

Comment:
Are we ok with keeping the different setting names `ConnectionUpdateInterval` in the operator and `HeartbeatInterval` in FLP?

Reply:
Makes sense to be more aligned. Would `ConnectionHeartbeatInterval` work ?

Reply:
Sure!

I wonder whether I should have renamed it to `ConnectionHeartbeatInterval` instead of `HeartbeatInterval` in FLP... 😅

Reply:
In `conntrack` stage it makes sense; it's less obvious in the CRD as we are in the `processor` config here